packages/playground/src/components/vm_deployment_table.vue

```diff
@@ -16,6 +16,16 @@
           <v-card-title style="color: #ffcc00; font-weight: bold">Failed Deployments</v-card-title>
           <v-divider color="#FFCC00" />
           <v-card-text>
+            Failed to load <strong>{{ count - items.length }}</strong> deployment{{
+              count - items.length > 1 ? "s" : ""
+            }}.
+
+            <span>
+              This might happen because the node is down or it's not reachable
+              <span v-if="showEncryption"
```

Comment:
is `showEncryption` always false? I can't find any assign/update for it's value anywhere

Reply:
condition added for `showEncryption`